scene/main/viewport.cpp

```diff
@@ -1461,6 +1461,8 @@ void Viewport::_gui_show_tooltip() {
 	panel->set_flag(Window::FLAG_NO_FOCUS, true);
 	panel->set_flag(Window::FLAG_POPUP, false);
 	panel->set_flag(Window::FLAG_MOUSE_PASSTHROUGH, true);
+	panel->set_flag(Window::FLAG_TRANSPARENT, true);
+	// A non-embedded tooltip window will only be transparent if per_pixel_transparency is allowed in the main Viewport.
```

Comment:
This comment should be above the flag change.
Is this related to what this PR fixes though?

Reply:
It's true that the PR would work fine without this, but you could not have transparent backgrounds in the non-embedded tooltips. Should I remove this and at some point do a separate PR as a bug-fix?

My sense is that because this PR makes non-embedded tooltips workable (because you couldn't use them previously for projects on HiDPI displays), no one would have come accross this bug.

Reply:
This seems to have caused a regression where editor tooltips are transparent unexpectedly: https://github.com/godotengine/godot/issues/92712#issuecomment-2261752610

Reply:
> A non-embedded tooltip window will only be transparent if per_pixel_transparency is allowed in the main Viewport.

It's allowed in the project settings not for a Viewport, and only affect requested surface format on window creation, but nothing prevents GPU driver from always using surface with alpha channel. And seems like Windows DisplayServer do not have any additional checks (unlike X11 and macOS) and will try to enable it regardless of project settings.